tests/AuthFactoryTest.php

```diff
@@ -29,6 +29,10 @@ public function testNewAuthWithSessionAndSegment()
 
     public function testNewPdoAdapter_passwordVerifier()
     {
+        if (false === extension_loaded('pdo_sqlite')) {
```

Comment:
How about we do a refactor to a trait ? 

$this->isExtensionLoaded('extension') ? and will it mark the test as skipped for it ?

In that case we could have saved some code.

Thoughts ?